test/16.js

```diff
@@ -0,0 +1,21 @@
+const expect = require('chai').expect;
+const duplicates = require('../solutions/16');
+const hasDuplicates = duplicates.hasDuplicates;
+
+describe('check duplicates', () => {
+  it('should return true since array has duplicates', () => {
+    const array = [1,2,3,3,4];
+    const expected = true;
+    expect(hasDuplicates(array)).eql(expected);
```

Comment:
chai should also have a .to.be.truthy or something like that method that checks for truthiness